src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -455,19 +458,36 @@ private function loadSymfony(ContainerBuilder $container, XmlFileLoader $loader,
         $loader->load('symfony.xml');
     }
 
+    private function loadCloudflare(ContainerBuilder $container, XmlFileLoader $loader, array $config)
+    {
+        $this->createHttpDispatcherDefinition(
+            $container,
+            array_merge(['servers' => ['https://api.cloudflare.com']], $config['http']),
```

Comment:
i wonder if we should allow to configure the domain? not sure if there is a use case for it in testing or such, but it feels a bit odd to hardcode it here.

if we do, we should of course provide this domain as default value so it only needs to be set when something else is wanted.